kokoro/linux/32-bit/build.sh

```diff
@@ -10,8 +10,9 @@
 # Change to repo root
 cd $(dirname $0)/../../..
 
-export DOCKERFILE_DIR=kokoro/linux/32-bit
+export DOCKERHUB_ORGANIZATION=protobuftesting
+export DOCKERFILE_DIR=kokoro/linux/dockerfile/test/php_32bit
 export DOCKER_RUN_SCRIPT=kokoro/linux/pull_request_in_docker.sh
 export OUTPUT_DIR=testoutput
 export TEST_SET="php_all_32"
-./kokoro/linux/build_and_run_docker.sh
+./kokoro/linux/build_and_run_docker2.sh
```

Comment:
Is this necessary? How about we just update build_and_run_docker to set a default DOCKERHUB_ORGANIZATION only if it is not already set? Then we don't need two almost identical script.